utils/escape.go

```diff
@@ -99,3 +100,27 @@ func Escape(tv string) string {
 	}
 	return result.String()
 }
+
+// Unescape takes string that have been escape to comply graphite's storage format
```

Comment:
It it's Graphite it should probably not be in this file.

Reply:
Done